### PR TITLE
Remove plugins, update some configs

### PR DESCRIPTION
**Pull request by nuriel77**
@IoTMOD I thought I would have enthusiastically manage to update everything, but that seems like a lot of work (e.g. tipsel options).

I tried to update some config options to have the docs kept more up-to-date.

In this PR you will see some descriptions where I've written `???` -> could you please provide me the text to add in those so I can update the PR?

Thanks!

### CODE REVIEW

**Comment by IoTMOD** [comment]
Hi @nuriel77,

thanks for your work, I'll have a look and provide the missing info later today 👍

File: Configuration.md [hunk 1]
```diff
@@ -104,11 +102,17 @@ You can adapt HORNET to your needs. The configuration options are shown below.
     ```json
     60
     ```
-- `coordinator.checkpointTransactions` int
-  - The amount of checkpoints issued between two milestones
+- `coordinator.checkpoint.maxTrackedTails` int
```
Comment (IoTMOD) [inline Configuration.md:105]: Please remove, it's `coordinator.checkpoint.maxTrackedTransactions` in the config (see below)

File: Configuration.md [hunk 2]
```diff
@@ -104,11 +102,17 @@ You can adapt HORNET to your needs. The configuration options are shown below.
...
+  - The amount of ???
     <br>Default:
     ```json
-    5
+    10000
+    ```
+- `coordinator.checkpoint.maxTrackedTransactions` int
+  - The amount of ???
```
Comment (IoTMOD) [inline Configuration.md:112]: `The maximum amount of known bundle tails for milestone tipselection`

File: Configuration.md [hunk 3]
```diff
@@ -694,6 +598,13 @@ You can adapt HORNET to your needs. The configuration options are shown below.
     ```json
     "HORNET99INTEGRATED99SPAMMER999999999999999999999999999999999999999999999999999999"
     ```
+- `spammer.autostart` bool
+  - ???
```
Comment (IoTMOD) [inline Configuration.md:602]: `Whether to automatically start the spammer on node startup`

**Review by muXxer — APPROVED** [review]

**Pull request merged**